.github/workflows/veracode-dast-scan.yml

```diff
@@ -0,0 +1,39 @@
+name: Veracode Dynamic Analysis Scan
+
+concurrency: ${{ github.ref }}-${{ github.workflow }}
+
+on:
+  push:
```

Comment:
@magile32 - can we run this on a schedule uising the cron feature in GHA? I'd imagine we want the DAST scan to either be a manual run or on main every morning.

```yml
Schedule: 
 -cron: '* * * * *'
```

Reply:
Set this to run M,W,F at 10PM EST